src/app.js

```diff
@@ -1,17 +1,5 @@
 const quotes = require("./quotes");
 const { getRandomElement } = require("./utils");
-
-const express = require("express");
-const app = express();
-const port = process.env.PORT || 3000;
```

Comment:
This is still needed here - app.js should be where the app is created, connected to the router and told to listen on the specified port.